### PR TITLE
Shellvars: allow command-specific environment variables

### PR DESCRIPTION
**Pull request by kunkku**

### CODE REVIEW

File: lenses/shellvars.aug [hunk 1]
```diff
@@ -157,7 +157,9 @@ module Shellvars =
     in let pipe = action_pipe (entry_eol_item command | entry_item command)
     in let and = action_and entry
     in let or = action_or entry
-    in Util.indent . label "@command" . store (word - reserved_key)
+    in Util.indent . label "@command"
+     . [ key key_re . eq . store anyquot . Sep.space . del /(\\\\\n[ \t]*)?/ "" ]*
```
Comment (raphink) [inline lenses/shellvars.aug]: For clarity, could you make this a named sublens (e.g. `envvar`)?

**Comment by raphink** [comment]
This looks good, but needs to be rebased.

**Comment by kunkku** [comment]
Now rebased and written using sublens. I also put Sep.cl_or_space here to make it cleaner.

**Comment by raphink** [comment]
Merged as 6193edbcf35e028800140a6cb6ce9692bdfbc99d

**Pull request closed without merging**